kagen/context.cpp

```diff
@@ -342,6 +342,12 @@ PGeneratorConfig CreateConfigFromString(const std::string& options_str, PGenerat
     config.coordinates = get_bool_or_default("coordinates");
     config.permute     = get_bool_or_default("permute");
 
+    // edge weight generation
+    const std::string edge_weight_generator_name = get_string_or_default("edgeweight-generator", "none");
```

Comment:
other options use `_` instead of `-`; I agree that `-` is a better choice since its also used for the CLI arguments; maybe accept both here, i.e., `get_string_or_default("edgeweight-generator", get_string_or_default("edgeweight_generator"))` etc; or chance the parser to make `-` and `_` interchangeable?

Reply:
I now replace every occurence of `-` with `_` in each key.